### PR TITLE
feat: Products page

### PR DESCRIPTION
**Pull request by joshua-mo-143**
*    ~~Adds a general Products page~~ Omitted due to PR scope change

 *   ~~Limits how many products are shown on the front page (and adds a "View all products" button that leads to the Products page)~~ Omitted due to PR scope change

*   Fixed error with stop-color instead of stopColor

*    Added extra tags to mock data for future experimental product search testing

*    Moved dev-only deps to devdeps.

Ideally it'd be great to have some kind of search filter function on the Products page but not sure if/when I have time to do this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+package-lock.json
```
Comment (Surf-N-Code) [inline .gitignore:37]: Why did you add the package lock to the ignore file?

Without it, everyone installing dependencies might get different versions to you or anyone else.

File: app/page.tsx [hunk 2]
```diff
@@ -39,5 +43,11 @@ export default function Home() {
           </ul>
         </div>
     </section>
+    <section className="max-w-4xl py-12 mx-auto px-4 md:px-8">
```
[the remark this answers is not in the document]
Comment (joshua-mo-143) [reply app/page.tsx:46]: Probably. I was experimenting with adding eslint but was thinking it would probably be better if it was left to you guys to do as I don't know what your preferred options are.

**Comment by joshua-mo-143** [comment]
Scope of PR changed to only moving dev dependencies.

**Comment by sidiDev** [comment]
@joshua-mo-143 Hey can you please handle the conflicts part?

**Comment by joshua-mo-143** [comment]
I'm going to close this PR - it'd be quicker and easier to just re-fork the whole thing again. Git isn't letting me fix the merge conflicts in the command line.
